runtimeconfig/google/cloud/runtimeconfig/variable.py

```diff
@@ -0,0 +1,197 @@
+
+    See:
+    https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/v1beta1/projects.configs.variables
+
+    :type name: string
+    :param name: The name of the variable.  This corresponds to the
+                 unique path of the variable in the config.
+
+    :type config: :class:`google.cloud.runtimeconfig.config.Config`
+    :param config: The config to which this variable belongs.
+    """
+
+    def __init__(self, name, config):
+        self.name = name
+        self.config = config
+        self._properties = {}
+
+    @property
+    def full_name(self):
+        """Fully-qualified name of this variable.
+
+        Example:
+        ``projects/my-project/configs/my-config/variables/my-var``
+
+        :rtype: string
+        :returns: The full name based on config and variable names.
+
+        :raises: :class:`ValueError` if the variable is missing a name.
+        """
+        if not self.name:
+            raise ValueError('Missing variable name.')
+        return '%s/variables/%s' % (self.config.full_name, self.name)
+
+    @property
+    def path(self):
+        """URL path for the variable's APIs.
+
+        :rtype: string
+        :returns: The URL path based on config and variable names.
+        """
+        return '/%s' % (self.full_name)
+
+    @property
+    def client(self):
+        """The client bound to this variable."""
+        return self.config.client
+
+    @property
+    def value(self):
+        """Value of the variable, as bytes.
+
+        See:
+        https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/v1beta1/projects.configs.variables
+
+        :rtype: bytes or ``NoneType``
+        :returns: The value of the variable or ``None`` if the property
+                  is not set locally.
+        """
+        if 'value' in self._properties:
+            return base64.b64decode(self._properties['value'])
+        return None
+
+    @property
+    def state(self):
+        """Retrieve the state of the variable.
+
+        See:
+        https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/v1beta1/projects.configs.variables#VariableState
+
+        :rtype: string or ``NoneType``
+        :returns: If set, one of "UPDATED", "DELETED", or
+                  "VARIABLE_STATE_UNSPECIFIED", else ``None``.
+        """
+        return self._properties.get('state')
+
+    @property
+    def update_time(self):
+        """Retrieve the timestamp at which the variable was updated.
+
+        See:
+        https://cloud.google.com/deployment-manager/runtime-configurator/reference/rest/v1beta1/projects.configs.variables
+
+        :rtype: :class:`datetime.datetime` or ``NoneType``
+        :returns: Datetime object parsed from RFC3339 valid timestamp, or
+                  ``None`` if the property is not set locally.
+        """
+        value = self._properties.get('updateTime')
+        if value:
```

Comment:
Use an explicit `None` check instead: `if value is not None`

From [PEP8][1]

> Also, beware of writing `if x` when you really mean `if x is not None` -- e.g. when testing whether a variable or argument that defaults to `None` was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!

[1]: https://www.python.org/dev/peps/pep-0008/#programming-recommendations